### ISSUE TITLE
Ban 

### ISSUE THREAD

**Issue by JON-GAMES** [hunterjm/futgui#174]
Has anyone received this Ban?
KNOW IF IT'S DEFINITIVE?

Hello, 

We noticed that your FIFA Account broke EA’s gameplay rules… 

We need to talk… 
Breaking our rules damages the experience for you and other players. We want to keep everyone’s gaming experience safe, secure, and enjoyable. 

You break our rules if you: 

Interfere with or disrupt any EA Service or any server or network used to support or provide an EA Service, including any hacking or cracking into an EA Service.
Use any software or program that damages, interferes with or disrupts an EA Service or another’s computer or property, such as denial of service attacks, spamming, hacking, or uploading computer viruses, worms, Trojan horses, cancel bots, spyware, corrupted files and time bombs.
Attempt to obtain, or phish for, a password, account information, or other private information from anyone else on EA Services.
Use any robot, spider or other automated device or process to access this website for any purpose or copy any material on this website.
Use or distribute unauthorized software programs or tools, such as “auto” software programs, “macro” software programs, “cheat utility” software program or applications, exploits, cheats, or any other game hacking, altering or cheating software or tool
Use exploits, cheats, undocumented features, design errors or problems in an EA Service.
Sell, buy, trade or otherwise transfer your EA Account, any personal access to EA Services, or any EA Content associated with your EA Account, including EA Virtual Currency and other Entitlements, either within an EA Service or on a third party website, unless expressly authorized by EA.
Promote, encourage or take part in any prohibited activity described above.
What this means for you 

We noticed some abnormalities within your account. Because of this, we have permanently banned you from online play. 
Read up on how to play by the rules at http://help.ea.com/en-us/help/faq/play-by-the-ea-rules
Review our full Terms of Service at http://www.ea.com/terms-of-service

Any questions? Contact us at help.ea.com. If you think we sent this to you by mistake, let us know at lets_talk@ea.com.
 

Thank you,

EA Terms of Service

thank you

**Comment by mestesso93** [comment]
yes, is a permanent ban. The reason is an high level of RPM, constant RPM (if you're human is impossible make always (for example) 12 request per minute. Probably you made a variable request, not always 12), and many time of bot (is impossible for human make bid 24/24h). You must set 4/5/6 RPM for 3/4 hours. And change RPM every hour. I receive 2 ban but now with these settings not.

**Comment by JON-GAMES** [comment]
So I totally lost my account. Do not I have a chance to get it back? Do you step through this?

**Comment by celioreyes** [comment]
@mestesso93 seems like it's gone. Unless you can somehow have EA remove the ban. I don't think @hunterjm can help. He just provided the tool, you used it at your own risk like all of us have.

**Comment by JON-GAMES** [comment]
I understand rsrs
It's a part of this. I'm going to continue using this wonderful tool as soon as I can create a new account.

**Comment by hunterjm** [comment]
Unfortunately, we can't do anything about that account, but if you could provide the settings you used and how often you ran the auto bidder it would be useful information.

**Comment by JON-GAMES** [comment]
Yes, I always used RPM 15, an average of 8 players, I was running about 3 times a day, for about 1h - 1h30.
He rarely received error 500.
It was about 10,000 a day.
I will start from scratch, this time I will vary the RPM and decrease the execution time.
One thing I noticed was that the ban happened after I won good prizes on the weekend call for the first time. I do not know if it was just a coincidente.

**Comment by JON-GAMES** [comment]
Yes, I always used RPM 15, an average of 8 players, I was running about 3 times a day, for about 1h - 1h30.
He rarely received error 500.
It was about 10,000 a day.
I will start from scratch, this time I will vary the RPM and decrease the execution time.
One thing I noticed was that the ban happened after I won good prizes on the weekend ligue  for the first time. I do not know if it was just a coincidence

**Comment by mestesso93** [comment]
No, probably the reason is the player that you use. For example, if you use MUSA, the bidder make many offer and with 15 rpm all auction are won. This is impossible for human. And if the price of a common player going up, EA probably have a system that notice it. 15 RPM are too high. You must choose more rare player and not common player, and you must manually change the RPM in a low range. (If you want simulate a human bidding, you must lose some auction. So 4-5 RPM are perfect).

Maybe the solution is edit a bidder with an auto-random change RPM with an interval that you set, but for this, we need the help of @hunterjm . So, mrhunter, i have also notice that the emulation of ios and android make an error. Any suggestion? Thanks!

**Comment by hunterjm** [comment]
@mestesso93: Honestly I'm no longer actively maintaining this version of the autobuyer.  As soon as this weekend, I will be releasing a [new version](https://github.com/hunterjm/fifa-autobuyer/releases) and deprecating this version.  What I've been working on has a much nicer interface.  I think you will all like it.  We can think about adding that feature there if you would like to create an issue for it.

**Comment by JON-GAMES** [comment]
Of course, it will be an honor. I will use it and report any problems I find.
Already have Windows version available?
Thanks

**Comment by anthonymac** [comment]
hey @hunterjm just wanna say thanks for all the hard work. 
@JON-GAMES sucks to hear u got banned. Next time try also to reduce player list to 3-4 players. 
I have been using 3 - 4 players at a time w/ 10 RPM for 3 hours per session. no error 500 or temp bans as of yet.

**Comment by JON-GAMES** [comment]
I knew the risks.
This tool is excellent, very well developed by  @hunterjm .
I'm waiting to release the new AB version for windows.
@anthonymac Thanks for the tips, I'll be using it to avoid a new BAN.
